device_service.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+)
+
+type Device struct {
+	Id       int     `json:"id"`
+	Name     string  `json:"name"`
+	Interval int     `json:"interval"`
+	Value    float32 `json:"value"`
+}
+
+type DeviceService struct {
+	Database
+}
+
+func (service *DeviceService) createDevice(w http.ResponseWriter, r *http.Request) {
```

Comment:
In golang we tend to use short names (one, two letters) for receiver name: https://github.com/golang/go/wiki/CodeReviewComments#receiver-names